### PR TITLE
Minor code clean up in magnet controller

### PR DESCRIPTION
**Pull request by optimumtact**

### CODE REVIEW

**Review by Cyberboss — APPROVED** [review]

File: code/game/machinery/magnet.dm [hunk 1]
```diff
@@ -166,17 +166,6 @@
 	else
 		use_power = NO_POWER_USE
 
-
-		// Overload conditions:
-		/* // Eeeehhh kinda stupid
-		if(on)
-			if(electricity_level > 11)
-				if(prob(electricity_level))
-					explosion(loc, 0, 1, 2, 3) // ooo dat shit EXPLODES son
```
Comment (RemieRichards) [inline code/game/machinery/magnet.dm]: 💥

**Comment by RemieRichards** [comment]
We should put magnets into a ruin or something.
Like an aperture science ruin.

cos they sure as shit don't get used on station.

**Review by Cyberboss — APPROVED** [review]

**Comment by Tacolizard** [comment]
we have magnets?

**Comment by RemieRichards** [comment]
@Tacolizard the firing range in science, the target stake can be programmed to move around, it's attached to a magnet.

that is literally the only use of magnets in the entirety of SS13.

**Pull request merged**